yogstation/code/modules/clothing/donator.dm

```diff
@@ -189,11 +189,20 @@
 	name = "Inmate belongings box"
 	desc = "A criminals' belongings, returned upon their release. Just stop killing people."
 
-/obj/item/storage/box/boxta/PopulateContents() 
+/obj/item/storage/box/boxta/PopulateContents()
 	. = ..()
 	new /obj/item/clothing/neck/falcon(src)
 	new /obj/item/clothing/head/fedora/gtrim_fedora(src)
 
+/obj/item/storage/box/somecustodian
+	name = "piss colored box"
```

Comment:
Sus
Also would this not be "piss-colored box"

Reply:
> You changed the icon names in code but haven't changed the icon file to reflect Also the helmet sprite flickers

to the icon names yes I actually did rename them

Reply:
> Sus Also would this not be "piss-colored box"

Its probably fine as is but I will change it

Reply:
![](https://i.imgur.com/J7cwJtc.png)

Reply:
![image](https://user-images.githubusercontent.com/62126495/220969399-d986c9c4-1ac1-4cb8-ad65-7aa72e23b22a.png)